packages/core/src/codewhisperer/activation.ts

```diff
@@ -594,6 +592,8 @@ export async function activate(context: ExtContext): Promise<void> {
         )
     }
 
+    void FeatureConfigProvider.instance.fetchFeatureConfigs()
```

Comment:
why not await this? if the intention is to do the work asynchronously, use `.catch()` so that failures are at least logged

Reply:
I will add a catch so that even if the API is down the extension is not impacted in the critical path.